src/2023/code-of-conduct/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Helvetic Ruby - A Ruby conference in Switzerland">
```

Comment:
Would something like "Code of conduct for the Helvetic Ruby conference" describe the page more precisely?

Reply:
Absolutely right. 👍